macros/tuva_invocations/drop_old_tuva_invocations.sql

```diff
@@ -0,0 +1,8 @@
+{% macro drop_old_tuva_invocations() %}
```

Comment:
Can we make this optional with a variable time period? I could see some people wanting to keep this log for longer than 7 days, especially for datasets that don't refresh that often.